commands/api.go

```diff
@@ -290,7 +293,26 @@ func apiCommand(cmd *Command, args *Args) {
 			}
 		}
 		if requestLoop && !parseJSON {
-			fmt.Fprintf(out, "\n")
+			_, _ = fmt.Fprintf(out, "\n")
+		}
+
+		if requestLoop {
+			var rateLimitLeft int
+			var rateLimitResetMs int
+			var atoiErr error
+			if xRateLimitLeft := response.Header.Get("X-Ratelimit-Remaining"); len(xRateLimitLeft) != 0 {
+				rateLimitLeft, atoiErr = strconv.Atoi(xRateLimitLeft)
+				utils.Check(atoiErr)
+			}
+			if xRateLimitReset := response.Header.Get("X-Ratelimit-Reset"); len(xRateLimitReset) != 0 {
+				rateLimitResetMs, atoiErr = strconv.Atoi(xRateLimitReset)
+				utils.Check(atoiErr)
+			}
+			if rateLimitLeft <= 1 {
```

Comment:
This variable is getting initialized with "0" as default, and here the condition is `<= 1`, which will always match if the server doesn't send the `X-Ratelimit-Remaining` header. This might be what's breaking the tests in CI.

Reply:
I actually did wonder if there was a circumstance under which the headers didn't come back, but you're right, it's a much more sane thing to just initialize the variable. I'll get that straightened out, too